.circleci/config.yml

```diff
@@ -348,6 +346,16 @@ jobs:
       - run: *test-salesforce-analytics
       - run: *test-salesforce-sdk
       - run: *test-smart-store
+      - run: *test-rest-explorer
+      - run:
```

Comment:
`SmartSync`, `SalesforceHybrid` and `SalesforceReact` are our longest running test suites. Restarting the emulator between these runs in an attempt to increase stability of the nightly runs.